boa_runtime/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+//! Example runtime for Boa
+//!
+//! This crate contains an example runtime for the `boa_engine` crate, so that it can be used as a
```

Comment:
Maybe this is a larger discussion, but should we be referring to this as an example runtime or as boa's basic runtime implementation and encourage implementors to use it as a template/build on it for their own needs?

Reply:
It only has one builtin for now, so I'm not sure at this point 😅 but I'm open to it, of course

Reply:
Neither am I lol I mainly mention it because the crate is named `boa_runtime` but it only has console in it. Since there have been issues submitted around a `setTimeout` implementation, someone may see the crate and immediately think: "oh, it's a runtime" and then get confused. If you're fine with it, then I'm good to merge this.

Reply:
For now, I would say, let's merge it, and add timeout and all this stuff to it slowly, see how it evolves :)